### DISCUSSION TITLE
clion 调试 doris be

### DISCUSSION THREAD

**Discussion by Cherry-Puppy** [apache/doris#10031] [A - General / Q&A]
哪位大佬用过 clion 调试过 be呢？

**Comment by dataalive** [comment]
这个具体有什么问题来着？

**Reply by Cherry-Puppy** [reply]
@dataalive hello 我在att 到 be pid 的时候出现死锁
<img width="1082" alt="image" src="https://user-images.githubusercontent.com/46736583/175875593-c3cabf4d-7a80-4e90-b755-d8520628e4d4.png">
